features/nationaliteit.feature

```diff
@@ -29,146 +29,138 @@ Functionaliteit: Bepalen van de actuele nationaliteit van een persoon
 
     @gba
     Scenario: de persoon wordt behandeld als Nederlander
-      Gegeven de persoon met burgerservicenummer '000009866' heeft de volgende gegevens
-      | Stapel | Categorie | nationaliteit (05.10) | reden opnemen (63.10) | reden beëindigen (64.10) | bijzonder Nederlanderschap (65.10) | onjuist (84.10) | datum ingang geldigheid (85.10) |
-      | 1      | 4         |                       | 310                   |                          | B                                  |                 | 19570115                        |
+      Gegeven de persoon met burgerservicenummer '000000024' heeft een 'nationaliteit' met de volgende gegevens
+      | reden opnemen (63.10) | bijzonder Nederlanderschap (65.10) | datum ingang geldigheid (85.10) |
+      | 310                   | B                                  | 19570115                        |
       Als personen wordt gezocht met de volgende parameters
       | naam                | waarde                              |
       | type                | RaadpleegMetBurgerservicenummer     |
-      | burgerservicenummer | 000009866                           |
-      | fields              | burgerservicenummer,nationaliteiten |
-      Dan heeft de persoon met burgerservicenummer '000009866' een nationaliteit met alleen de volgende gegevens
+      | burgerservicenummer | 000000024                           |
+      | fields              | nationaliteiten |
+      Dan heeft de response een persoon met een 'nationaliteit' met de volgende gegevens
       | naam                                | waarde   |
```

Comment:
hier verwacht je wel GbaPersoon output. had je deze dan niet ook moeten omschrijven naar een end-2-end scenario met dus proxy output?

Reply:
Nu de hele "nationaliteiten" proxy-output beschreven in de Dan stap vanwege end-to-end testen.